app/scripts/modules/netflix/fastProperties/domain/manualJudgementStage.ts

```diff
@@ -14,8 +14,8 @@ export class ManualJudgementStage implements IStage {
   instructions: string = 'Is Fast Property good to move forward?';
   propagateAuthenticationContext: boolean = true;
 
-
-  constructor(user: IUser, command: PropertyCommand, previousStage?: IStage) {
+  // TODO: ANG, talk with Zthrash about use of command arg.
```

Comment:
my thought was that there were a lot of files that needed changing and I didn't want to lose them.